include/zephyr/bluetooth/mesh/access.h

```diff
@@ -31,6 +35,8 @@
 #define BT_MESH_MODEL_UUIDS_UNASSIGNED()
 #endif
 
+/** @endcond */
```

Comment:
I see that we some places do

```suggestion
/** INTERNAL_HIDDEN @endcond */
```

or

```suggestion
/** @endcond INTERNAL_HIDDEN */
```

To help indicate what this condition refers to

Reply:
@Thalley right, probably something we should generalize. Note that the second option is incorrect though, since `@endcond` doesn't take any argument :)

```
/** @endcond INTERNAL_HIDDEN */
```

so there's definitely a few offenders that would need fixing: https://docs.zephyrproject.org/latest/doxygen/html/group__modbus.html#gaa0acc60971d0f773719431173632fd30